tests/foreman/ui/test_operatingsystem.py

```diff
@@ -176,7 +161,7 @@ def test_positive_verify_os_name(session):
     name = gen_string('alpha')
     major_version = gen_string('numeric', 2)
     os_full_name = f"{name} {major_version}"
-    entities.OperatingSystem(name=name, major=major_version).create()
+    target_sat.api.OperatingSystem(name=name, major=major_version).create()
     with session:
         values = session.operatingsystem.search(os_full_name)
         assert values[0]['Title'] == os_full_name
```

Comment:
Can this test to search w/ full name also be combined with above test?

Reply:
Yes, but I'll prefer to keep it as while creating OS here we only pass name and major version so the name parameter comes out as `f"{name} {major_version}"`  but in the end-to-end test we pass all the fields  including descriptionso it's different